src/lib.rs

```diff
@@ -94,28 +95,34 @@ impl Cloudinary {
         }
     }
 
-    /// uploads an image
+    /// upload files of type `ResourceTypes`
     /// ```rust
     /// use cloudinary::{Source, Cloudinary};
     /// use cloudinary::upload::{UploadOptions};
-    /// let options = UploadOptions::new().set_public_id("file.jpg".to_string());
-    /// let cloudinary = Cloudinary::new("api_key".to_string(), "cloud_name".to_string(), "api_secret".to_string() );
-    /// let result = cloudinary.upload_image(Source::Path("./image.jpg".into()), &options);
+    /// use upload::resource_type::ResourceTypes;
+    /// let cloudinary = Cloudinary::new(
+    ///         "api_key".to_string(),
+    ///         "cloud_name".to_string(),
+    ///         "api_secret".to_string()
+    ///     );
+    /// let options = UploadOptions::new().set_public_id("app_data.sql".to_string());
```

Comment:
Let's use sane examples.

Reply:
I actually did plan on saving some sql files to the cloud 🙂

Reply:
Yeah, but you're passing it to a `cloudinary.upload_image`

Reply:
Oh sorry about that. I've fixed the examples.